rocketpy/mathutils/function.py

```diff
@@ -1,9 +1,11 @@
 from inspect import signature
```

Comment:
Please use [isort](https://pycqa.github.io/isort/) to sort the imports in a .py file.

Reply:
Done